src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js

```diff
@@ -269,7 +269,11 @@ import { getLowerAgeYears } from '@ncigdc/utils/ageDisplay';
     }
 
     self.legend_id = `legend_${self._name}`;
-    self.legend_continuous_categories = ['age_at_diagnosis', 'days_to_death'];
+    self.legend_continuous_categories = [
+      'age_at_diagnosis',
+      // 'days_to_death'
+      // hide days to death from legend
+    ];
```

Comment:
Wish we could have the data tell us which features are meant to be continuous and which categorical, so we wouldn't have to be hardcoding values everywhere.